bundles/core/src/test/java/com/adobe/cq/commerce/core/components/internal/models/v1/button/ButtonImplTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2019 Adobe.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adobe.cq.commerce.core.components.internal.models.v1.button;
+
+import org.apache.sling.api.scripting.SlingBindings;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.adobe.cq.commerce.core.components.models.button.Button;
+import com.adobe.cq.sightly.WCMBindings;
+import com.day.cq.wcm.api.Page;
+import io.wcm.testing.mock.aem.junit.AemContext;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
What is this required for?

Reply:
It is used to initialise the object with Mock annotation.But now instead of annotation ,as i am using the mock method to initialise the Page object, its not required.

Reply:
This is fixed